projects/ng-imgix/package.json

```diff
@@ -8,5 +8,67 @@
   "dependencies": {
     "imgix-core-js": "2.3.1",
     "urijs": "1.19.2"
+  },
+  "release": {
+    "branches": [
+      "main",
+      { "name": "next", "prerelease": "rc" },
+      { "name": "beta", "prerelease": true },
+      { "name": "alpha", "prerelease": true }
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+
+      [
+        "@google/semantic-release-replace-plugin",
+        {
+          "replacements": [
+            {
+              "files": [
+                "src/lib/ix-srcset.directive.ts"
+              ],
+              "from": "VERSION = '.*'",
+              "to": "VERSION = '${nextRelease.version}'",
+              "results": [
+                {
+                  "file": "src/lib/ix-srcset.directive.ts",
+                  "hasChanged": true,
+                  "numMatches": 1,
+                  "numReplacements": 1
+                }
+              ],
+              "countMatches": true
+            }
+          ]
+        }
+      ],
+
+      "@semantic-release/changelog",
+      "@semantic-release/npm",
+      [
+        "@semantic-release/git",
+        {
+          "assets": [
+            "src/**",
+            "../../dist/ng-imgix/**",
```

Comment:
What happens when the actual npm package is published? It seems like the dist files should probably moved under the `ng-imgix/dist` directory?

Reply:
This was surprisingly [more confusing](https://github.com/angular/angular-cli/issues/12979) to achieve than I originally expected 😡